requirements.txt

```diff
@@ -6,3 +6,4 @@ numpy>=1
 pandas>=1
 requests>=1
 pillow>=5.3.0
+pydicom>=2.3.1
```

Comment:
There was an issue when requiring this for the entire library for some users. It should be optional and then if someone is using dicom then there will be an import error and they can install it.

Reply:
Do you think it will be better to have a `requirements-dev.txt` along the lines of this https://github.com/Project-MONAI/MONAI/blob/dev/requirements-min.txt and can stay in the main directory?

Reply:
Ya I like it.